src/rinterface_extra.c

```diff
@@ -8331,6 +8331,59 @@ SEXP R_igraph_incident_edges(SEXP pgraph, SEXP pe, SEXP pmode) {
   return result;
 }
 
+SEXP R_igraph_power_law_fit_new(SEXP data, SEXP xmin, SEXP force_continuous, SEXP compute_pvalue)
+{
+  igraph_vector_t c_data;
+  igraph_plfit_result_t c_res;
+  igraph_real_t c_xmin;
+  igraph_bool_t c_force_continuous, c_compute_pvalue;
+  SEXP result, names;
+
+  SEXP r_result;
+
+  R_SEXP_to_vector(data, &c_data);
+  IGRAPH_R_CHECK_REAL(xmin);
+  c_xmin = REAL(xmin)[0];
+  IGRAPH_R_CHECK_BOOL(force_continuous);
+  c_force_continuous = LOGICAL(force_continuous)[0];
+  IGRAPH_R_CHECK_BOOL(compute_pvalue);
+  c_compute_pvalue = LOGICAL(compute_pvalue)[0];
+
+  IGRAPH_R_CHECK(igraph_power_law_fit(&c_data, &c_res, c_xmin, c_force_continuous));
```

Comment:
```suggestion
  # Can't use the generated `R_igraph_power_law_fit()` because we need `c_res` to compute the p-value
  IGRAPH_R_CHECK(igraph_power_law_fit(&c_data, &c_res, c_xmin, c_force_continuous));
```